src/components/dialog/bl-dialog.test.ts

```diff
@@ -301,6 +301,16 @@ describe('bl-dialog', () => {
           expect(ev.detail.isOpen).to.equal(false);
         });
       });
+
+      it('should not fire bl-dialog-close event when dialog open prop is false in default', async () => {
+        const el = await fixture<typeOfBlDialog>(html`<bl-dialog :open="false" caption="My title">
```

Comment:
`:open` doesn’t mean same with `lit-html` as `vuejs`. Basically here you are setting a property named `:open` with value of string `false`.  Actually there is no way to set a boolean attribute to `false` in HTML because attribute values are always string. As “boolean attribute” we mean just existence of an attribute. So `<bl-dialog open>` means a `true` value for `open`. If you don’t set `open`, it’ll be false. So `open=“false”` still means a truety value for `open` property. But you can set property instead of attribute as `.open=${false}`. This sets the property of input class  with a real `false` value, instead of attribute.

Reply:
Thanks a lot for this information. 
I'm new to web components and this is my very first PR in Baklava 🙏